cmd/buildah/manifest.go

```diff
@@ -101,6 +102,11 @@ func init() {
 	if err := flags.MarkHidden("arch"); err != nil {
 		panic(fmt.Sprintf("error marking --arch as hidden: %v", err))
 	}
+	flags.BoolVar(&manifestCreateOpts.insecure, "insecure", false, "neither require HTTPS nor verify certificates when accessing the registry. TLS verification cannot be used when talking to an insecure registry.")
```

Comment:
soft suggestion for consideration
```suggestion
	flags.BoolVar(&manifestCreateOpts.insecure, "insecure", false, "neither require HTTPS nor verification of certificates when accessing the registry. TLS verification cannot be used when talking to an insecure registry.")
```

Reply:
Then down below too if you take it.

Reply:
Since it is hidden, it really does not matter, nobody will ever see it.